nginx-proxy-manager/docker-compose.yml

```
@@ -0,0 +1,22 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: nginx-proxy-manager_web_1
+      APP_PORT: 81
+  web:
+    image: jc21/nginx-proxy-manager:2.11.3@sha256:5bd2aae54ac0b6f4ef6777143ab736baf5366807894a4c7fca7bfa2fbaf1489c
+    hostname: nginx-proxy-manager_web_1
+    restart: on-failure
+    ports:
+      - 443:443
```

Nice job realizing that port 80 is used by umbrelOS 👌

What we can actually do here is map ports 80 and 443 from the `nginx-proxy-manager` container to different ports on the host. So for example:
```suggestion
    ports:
      - 40080:80
      - 40443:443
```

Then in the app description we could clearly state that users need to port forward 80 to 40080 and 443 to 40443 on their router.

This way we completely avoid current clashes with port 80 and future clashes with port 443. What do you think?

---

Yes, this  should be fine. Forwarding both 80 and 443 ports will make HTTP-01 challenge work as well.